spinnaker/spinnaker_system/kube_smoke_test.py

```diff
@@ -330,7 +496,7 @@ def main():
 
   defaults = {
       'TEST_STACK': 'tst',
-      'TEST_APP': 'kubsmok' + KubeSmokeTestScenario.DEFAULT_TEST_ID
+      'TEST_APP': 'kubfimg' + KubeSmokeTestScenario.DEFAULT_TEST_ID
```

Comment:
why fimg? I liked smok better since more obvious to track back resources to this test.